test/averages/mean-test.js

```diff
@@ -0,0 +1,33 @@
+const assert = require('chai').assert
+const mean = require('../../lib/averages/mean.js')
+
+describe('mean functionality', function() {
+  context('mean function', function(){
+    it('it can determine the average of an array of numbers', function(){
```

Comment:
you have a lot of use cases all within this it block. i suggest creating a different it block for each use case. for example...

```js
it('returns the average of an array of integers', function(){
...
}
it('returns null if the array is empty', function(){
...
}

```